CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - A check to `New-PuppetDscModule` which validates that PSRemoting is enabled and errors clearly if not in order to prevent unexpected execution failures later in the process ([#133](https://github.com/puppetlabs/Puppet.Dsc/issues/133))
 - Long file path support information to the requirements section of autogenerated READMEs ([#149](https://github.com/puppetlabs/Puppet.Dsc/issues/149))
 - Notes on installation limitations to generated README files to clarify minitar limitations for `r10k` and serverless Puppet users ([#150](https://github.com/puppetlabs/Puppet.Dsc/issues/150))
+- (GH-139) - Clarification added regarding the DSC Local Configuration Manager (LCM) ([#139](https://github.com/puppetlabs/Puppet.Dsc/issues/139))
```

Comment:
We can drop the ticket prepend at the start of the entry and reword to be explicit about the change.

```suggestion
- Clarification regarding configuring the DSC Local Configuration Manager (LCM) to work with Puppetized DSC modules ([#139](https://github.com/puppetlabs/Puppet.Dsc/issues/139))
```